cmd/chart-repo/utils.go

```diff
@@ -415,3 +426,45 @@ func chartTarballURL(r repo, cv chartVersion) string {
 	}
 	return source
 }
+
+func initNetClient(rootCACert, additionalCACertDir string) (*http.Client, *http.Transport, error) {
+	caCertPool := x509.NewCertPool()
+	systemCACert, err := ioutil.ReadFile(rootCACert)
```

Comment:
Can't we just load the default pool using `x509.SystemCertPool()`, falling back to `NewCertPool` if that comes back empty? See https://forfuncsake.github.io/post/2017/08/trust-extra-ca-cert-in-go-app/?